src/NuGet.Clients/NuGet.PackageManagement.UI/Options/AddMappingDialog.xaml.cs

```diff
@@ -0,0 +1,121 @@
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Windows;
+using System.Windows.Input;
+using Microsoft.Build.Utilities;
+using Microsoft.ServiceHub.Framework;
+using NuGet.PackageManagement.UI;
+using NuGet.PackageManagement.VisualStudio;
+using NuGet.VisualStudio.Common;
+using NuGet.VisualStudio;
+using NuGet.VisualStudio.Internal.Contracts;
+using NuGet.Configuration;
+using System.Collections.Generic;
+using Task = System.Threading.Tasks.Task;
+using Resx = NuGet.PackageManagement.UI.Resources;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.Internal.VisualStudio.PlatformUI;
+
+using Microsoft.VisualStudio.Shell;
+
+
+namespace NuGet.Options
```

Comment:
This namespace doesn't feel right to me. Usually project default namespaces are the project name + folder within the project, so shouldn't the namespace be `NuGet.PackageManagement.UI.Options`?

Same question for the other new files in this PR.

Reply:
I decided not to change the namespace since the two existing options pages are in the NuGet.Options namespace.